runtime/lua/vim/uri.lua

```diff
@@ -51,23 +51,31 @@ end
 
 
 --@private
-local function is_windows_file_uri(uri)
-  return uri:match('^file:///[a-zA-Z]:') ~= nil
+-- For test cases, uri is hard-coded.
+-- So control the return value through a variable parameter
+-- If we test a windows uri param is 0
+local function is_windows(...)
```

Comment:
I am not sure I understand what you mean here

Reply:
Ok .Sorry for my english.. In test file `test/functional/lua/uri_spec.lua`, we test windows uri by pass a hadcored uri test case ` exec_lua([[filepath = 'C:\\Foo\\Bar\\Baz.txt']])` , if we use a variable to judgment current os type will caused the test failed. So when we test windows uri we can pass a value into this function for test.

Reply:
Lets have @tjdevries opinion on this, maybe we want to only do those tests on actual windows platforms, and not on linux ?

Reply:
Then we don't need to pass in any parameters. Everything will also become cheaper.